crowdsourcehinter/crowdsourcehinter.py

```diff
@@ -96,10 +96,10 @@ def studio_view(self, context=None):
         frag.add_javascript_url('//cdnjs.cloudflare.com/ajax/libs/mustache.js/0.8.1/mustache.min.js')
         frag.add_css(self.resource_string("static/css/crowdsourcehinter.css"))
         frag.add_javascript(self.resource_string("static/js/src/crowdsourcehinter_studio.js"))
-        print type(self.initial_hints), type(self.target_problem), type(self.generic_hints), str
+        print type(self.initial_hints), type(self.target_problem), type(self.generic_hints)
```

Comment:
This was my error, but we shouldn't have prints in code at the stage it is on edx-platform.